src/scales/scale.radialLinear.js

```diff
@@ -47,10 +47,300 @@ module.exports = function(Chart) {
 		}
 	};
 
+	function getValueCount(scale) {
+		return !scale.options.lineArc ? scale.chart.data.labels.length : 0;
+	}
+
+	function getPointLabelFontOptions(scale) {
+		var pointLabels = scale.options.pointLabels;
+		var pointLabelFontSize = helpers.getValueOrDefault(pointLabels.fontSize, globalDefaults.defaultFontSize);
+		var pointLabelFontStyle = helpers.getValueOrDefault(pointLabels.fontStyle, globalDefaults.defaultFontStyle);
+		var pointLabelFontFamily = helpers.getValueOrDefault(pointLabels.fontFamily, globalDefaults.defaultFontFamily);
+		var pointLabelFont = helpers.fontString(pointLabelFontSize, pointLabelFontStyle, pointLabelFontFamily);
```

Comment:
Quite personal, but I'm not a huge fan of long variable names, especially when the scope is obvious: it makes the code harder to read.

```javascript
function getPointLabelFontOptions(scale) {
    var options = scale.options.pointLabels;
    var size = helpers.getValueOrDefault(options.fontSize, globalDefaults.defaultFontSize);
    var style = helpers.getValueOrDefault(options.fontStyle, globalDefaults.defaultFontStyle);
    var family = helpers.getValueOrDefault(options.fontFamily, globalDefaults.defaultFontFamily);

    return {
        size: size,
        style: style,
        family: family,
        font: helpers.fontString(size, style, family);
    };
```